natalie.swift

```diff
@@ -843,7 +843,7 @@ class StoryboardFile {
                             println("            }")
                             println("        }")
                             println()
-                            println("        var identifier: String { return self.description } ")
+                            println("        var identifier: String? { return self.description } ")
```

Comment:
Because `UIStoryboardSegue` already has an `identifier` property, which is optional.
As I wanted to make `UIStoryboardSegue` conform to `SegueProtocol`, this was the easiest way.